ci-operator/config/openshift/release/openshift-release-master__ci-4.15.yaml

```diff
@@ -232,18 +229,17 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-azure-sdn-serial
-- as: e2e-azure-sdn-techpreview-serial
+- as: e2e-azure-techpreview-serial
```

Comment:
Preference would be to keep the network type in the [job name](https://docs.ci.openshift.org/docs/how-tos/naming-your-ci-jobs/) as some code still relies on parsing the job names